tests/Doctrine/Tests/DBAL/Cache/QueryCacheProfileTest.php

```diff
@@ -77,67 +73,64 @@ public function testShouldGenerateAnAutomaticKeyIfNoKeyHasBeenGiven()
 
     public function testShouldGenerateDifferentKeysForSameQueryAndParamsAndDifferentConnections()
     {
-        $query  = 'SELECT * FROM foo WHERE bar = ?';
```

Comment:
Some in the diffs in the test seem completely wrong/unrelated. Also, as I mentioned before, a new method verifying that the connection parameters cannot be found in the key is needed.

Reply:
Yes, I tried to make it more dry!